### PR TITLE
Add celery

### PR DESCRIPTION
**Pull request by bloom1**

### CODE REVIEW

File: wazo_call_logd/controller.py [hunk 1]
```diff
@@ -91,9 +105,11 @@ def run(self):
         finally:
             logger.info('Stopping wazo-call-logd')
             self.bus_client.stop()
-            self.bus_publisher.stop()
+            self._publisher.stop()
```
Comment (fblackburn1) [inline wazo_call_logd/controller.py:108]: Probably a wrong rebase?

**Review by fblackburn1 — CHANGES_REQUESTED** [review]

File: wazo_call_logd/controller.py (hunk 1 above)
Comment (bloom1) [reply wazo_call_logd/controller.py:108]: Yes, I'm fixing it

**Review by fblackburn1 — APPROVED** [review]

**Comment by fblackburn1** [comment]
recheck

**Comment by bloom1** [comment]
> Build failed.
> 
>     * [wazo-tox-py37 ](https://sf.production.wazo.cloud/zuul/t/local/build/8faffb343ca645148d558dffa476df32) : SUCCESS in 2m 43s
> 
>     * [tox-linters ](https://sf.production.wazo.cloud/zuul/t/local/build/aa0e6f471fb34b07b33c519c6b0061c8) : SUCCESS in 2m 27s
> 
>     * [debian-packaging ](https://sf.production.wazo.cloud/zuul/t/local/build/20ed84a7bef7400ea1491fd2e1c0aad9) : SUCCESS in 2m 58s
> 
>     * [wazo-tox-integration-call-logd ](https://sf.production.wazo.cloud/zuul/t/local/build/8be38138baf94e53863bfd360bc57dbb) : FAILURE in 4m 22s

Sorry. I had the commit to fix the Dockerfile in another branch.

**Review by fblackburn1 — APPROVED** [review]

**Pull request merged**